packages/cli/src/WorkflowExecuteAdditionalData.ts

```diff
@@ -95,9 +95,8 @@ export function executeErrorWorkflow(
 
 	let pastExecutionUrl: string | undefined;
 	if (executionId !== undefined) {
-		pastExecutionUrl = `${WebhookHelpers.getWebhookBaseUrl()}workflow/${
-			workflowData.id
-		}/executions/${executionId}`;
+		const { editorBaseUrl } = Container.get(URLService);
+		pastExecutionUrl = `${editorBaseUrl}workflow/${workflowData.id}/executions/${executionId}`;
```

Comment:
changed this to use editor url instead of webhook url, as i believe that is the correct url.